### PR TITLE
Benjamin Bulb Problem

### PR DESCRIPTION
**Pull request by shivangi00**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
### Description
Submitting code for issue #1668 in folder /Java/other
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Java/other/BenjaminBulbs.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*Given n bulbs. At first, all of them are toggled.
+In the second go, every second bulb is toggled. 
+In the third go, every third bulb is toggled, and so on.
+Find the number of bulbs that will be ON after the process ends.
+*/
+
+import java.util.*;
+
+public class BenjaminBulbs{
+    public static void main(String args[]){
```
Comment (HarshCasper) [inline Java/other/BenjaminBulbs.java:10]: Shift all the primary Code Logic to another function. `main()` is primarily used for the driver code purpose.

File: Java/other/BenjaminBulbs.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/*Given n bulbs. At first, all of them are toggled.
+In the second go, every second bulb is toggled. 
+In the third go, every third bulb is toggled, and so on.
+Find the number of bulbs that will be ON after the process ends.
+*/
```
Comment (HarshCasper) [inline Java/other/BenjaminBulbs.java:5]: Add a link to the Problem because it looks like from some online Code Portal.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by atarax665** [comment]
@shivangi00 Are you still working on this?

**Comment by shivangi00** [comment]
> @shivangi00 Are you still working on this?

Yes I am.

**Comment by shivangi00** [comment]
I am creating a new PR for this with all needed changes.

**Pull request closed without merging**